src/core_plugins/kibana/public/management/sections/indices/create_index_pattern/create_index_pattern.html

```diff
@@ -76,7 +76,6 @@
                   class="kuiLink"
                   href="https://en.wikipedia.org/wiki/ISO_week_date"
                   target="_blank"
-                  title="Wikipedia: ISO Week Date"
```

Comment:
Out of curiosity: why is using `title` bad? Not as part of this PR but it might be good to include this guideline (with rationale) in our [HTML style guide](https://github.com/elastic/kibana/blob/master/style_guides/html_style_guide.md).